MedicMeditationApp/Presentation/SignIn/SignInView.swift

```diff
@@ -9,73 +9,89 @@ import SwiftUI
 
 struct SignInView: View {
     
+    @EnvironmentObject private var appRootManager: AppRootManager
+    
     @State var email: String = ""
     @State var password: String = ""
     
-    @State var isActiveMenu: Bool = false
-    @State var isActiveSignUp: Bool = false
+    var onClickLogin: () -> Void
+    var onClickSignUp: () -> Void
+    var onClickForgotPasword: () -> Void
     
     var body: some View {
-        NavigationView {
-            VStack {
-                ZStack {
-                    VStack(alignment: .leading) {
-                        Image(ImageResource.log)
-                        
-                        Text("Sign In")
-                            .font(.custom("AlegreyaSans-Medium", size: 30))
-                        Text("Sign in now to access your excercises")
-                            .font(.custom("AlegreyaSans-Regular", size: 22))
-                        Text("and save music.")
-                            .font(.custom("AlegreyaSans-Regular", size: 22))
-                        
-                        TextField("Email Address", text: $email)
-                            .padding()
-                            .cornerRadius(10)
-                        TextField("Password", text: $password)
-                            .padding()
-                            .cornerRadius(10)
-                        
-                        Button(action: {}, label: {
-                            Text("Forgot Password?")
-                                .font(.custom("AlegreyaSans-Regular", size: 14))
+        VStack {
+            ZStack {
+                VStack(alignment: .leading) {
+                    Image(ImageResource.log)
+                        .resizable()
+                        .frame(width: 80, height: 80)
+                        .padding(.bottom,20)
+                        .padding(.top,20)
+                    
+                    Text(L10n.SignIn.Title.text)
+                        .font(Fonts.AlegreyaSans.medium.swiftUIFont(size: 30))
+                    Text(L10n.SignIn.Subtitle.text)
+                        .font(Fonts.AlegreyaSans.regular.swiftUIFont(size: 22))
+                    Text(L10n.SignIn.SubtitleDos.text)
+                        .font(Fonts.AlegreyaSans.regular.swiftUIFont(size: 22))
+                    
+                    TextField(L10n.SignIn.Email.text, text: $email)
```

Comment:
Agregale porfa linea en el borde mira este link https://sarunw.com/posts/textfield-in-swiftui/#styling